pom.xml

```diff
@@ -52,4 +56,24 @@
         <module>targetplatform</module>
     </modules>
 
+
+    <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>com.itemis.maven.plugins</groupId>
+              <artifactId>unleash-maven-plugin</artifactId>
+              <version>2.6.0</version>
```

Comment:
Please extract the version number to a property like we do with other version numbers (at least in the `poms/pom.xml`)

Reply:
Why? Version number will only used at this single place. If there is no specific reason to do it different I think we should keep this information within the context it belongs to.

Reply:
The count is not the reason, we want them all together see for example: `build.helper.maven.plugin.version`

Reply:
...and to bring a counter examples: `license-maven-plugin`, `maven-bundle-plugin`, `maven-scr-plugin`, `saxon`. For all those it's explicitely defined. I wouldn't prefer to add a property for every single version declaration. But if we do it, we should do it consistently, shouldn't we?

Reply:
@kaikreuzer wdyt?

Reply:
@kaikreuzer ^^^

Reply:
Given you are in the process of cleaning it up, lets merge this. Sorry that I forgot about this PR for the past two weeks!

Reply:
No problem. I'll proceed on the weekend :)